extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/KafkaTuningConfig.java

```diff
@@ -288,76 +76,74 @@ public KafkaTuningConfig withBasePersistDirectory(File dir)
   }
 
   @Override
-  public boolean equals(Object o)
+  public KafkaTuningConfig copyOf()
```

Comment:
I don't understand what this method means. Probably we don't need this.

Reply:
Looks like it's only used for testing, will move this to tests